packages/front-end/components/Metrics/MetricForm.tsx

```diff
@@ -816,6 +825,43 @@ GROUP BY
             green, yellow, or red.
           </small>
         </div>
+        <div className="form-group">
+          Minimum threshold for significance
+          <input
+            type="number"
+            className="form-control"
+            {...inputs.minThresholdSignificance}
+          />
+          <small className="text-muted">
+            The number of events this metric requires before this metric will
+            show as significant. (default{" "}
+            {defaultMinConversionThresholdSignificance})
+          </small>
```

Comment:
For label and description, I might try:

**Minimum Sample Size**
The number of conversions required before doing a statistical analysis (default X)

Reply:
done